src/libs/kata-types/src/config/runtime.rs

```diff
@@ -17,14 +17,14 @@ pub const RUNTIME_NAME_VIRTCONTAINER: &str = "virt_container";
 #[derive(Debug, Default, Deserialize, Serialize)]
 pub struct Runtime {
     /// Runtime name: Plan to support virt-container, linux-container, wasm-container
-    #[serde(default)]
+    #[serde(default = "default_runtime")]
     pub name: String,
 
-    /// Hypervisor name: Plan to support dragonball, qemu
+    /// Hypervisor name: Plan to support dragonball, qemu, cloud hypervisor
     #[serde(default)]
     pub hypervisor_name: String,
 
-    /// Agent name
+    /// Agent name: Plan to support kata
     #[serde(default)]
     pub agent_name: String,
```

Comment:
I think `hypervisor_name` and `agent_name` need a default name.

Reply:
Either that, or we need to return an error of either of these is the empty string.